src/makeRobot.js

```diff
@@ -38,7 +38,66 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      const robotName = `name: ${this.name},`;
+      const robotVersion = ` chip version: ${this.version},`;
+      const robotWheels = ` wheels: ${this.wheels}`;
+
+      return robotName + robotVersion + robotWheels;
```

Comment:
Looks a bit weird to have commas/space in variables like this, I'd organize it like this
```suggestion
      const robotName = `name: ${this.name}`;
      const robotVersion = `chip version: ${this.version}`;
      const robotWheels = `wheels: ${this.wheels}`;

      return `${robotName}, ${robotVersion}, ${robotWheels}`;
```